composer/trainer/trainer.py

```diff
@@ -90,6 +90,8 @@ class contains metrics relevant to the specific dataset. Set to ``None`` for no
             or 'amp' (recommended). (default: ``Precision.FP32``).
         scale_schedule_ratio (float, optional): Ratio by which to scale the training duration and learning rate
             schedules. See :func:`scale_schedule` for details. (default: ``1.0``)
+        adaptive_grad_accum (boolean, optional): Dynamically scale down minibatch size and use gradient 
```

Comment:
Can just have the `grad_accum` parameter being set to `-1` used to signal autotuning